ts/Dashboards/Components/DataGridComponent/DataGridSyncs/DataGridHighlightSync.ts

```diff
@@ -58,21 +58,21 @@ const syncPair: Sync.SyncPair = {
 
         const { dataCursor: cursor } = board;
 
-        const onDataGridHover = (e: any): void => {
+        const onCellHover = (e: any): void => {
             const table = this.getFirstConnector()?.table;
             if (table) {
-                const row = e.row;
+                const cell = e.target;
 
                 cursor.emitCursor(table, {
                     type: 'position',
-                    row: parseInt(row.dataset.rowIndex, 10),
-                    column: e.columnName,
+                    row: parseInt(cell?.row?.index, 10),
+                    column: cell?.column?.id,
```

Comment:
What in the case when row/column will be undefined? IMO we should not call the emitCursor.